scripts/firedrake-install

```diff
@@ -301,6 +316,17 @@ def git_update(name):
         else:
             check_call(["git", "checkout", travis_commit])
     else:
+        # Ensure remotes get updated if and when we move repositories.
+        if url:
+            plain_url = split_requirements_url(url)[1]
+            current_url = check_output(["git", "remote", "-v"]).split()[1]
```

Comment:
This relies on `origin` being the only remote and someone not configuring a separate push and fetch urls.  I guess this is probably a sane default assumption.  If you wanted to separately find the push and fetch urls, guaranteeing that you did so for `origin`, you would do:

```
push_url = check_output(["git", "remote", "get-url", "--push", "origin"])
fetch_url = check_output(["git", "remote", "get-url", "origin"])
```